src/modules/savedQuery/saga/isQueryEditable.ts

```diff
@@ -0,0 +1,41 @@
+import { all, call, select } from 'redux-saga/effects';
+import { getEventStreams } from '../../schemas/selectors';
+import { fetchEventStreamProperties } from '../../schemas';
+import { checkIfStreamsExists } from './checkIfStreamsExists';
+
+export function* isQueryEditable(query) {
+  const eventStreamsToCheck = [];
+  let queryHasExistingEventStream;
```

Comment:
IMO the better naming could be just `allEventStreamsExist` - returning variable with this name will be self-descriptive. What do You think ?